### PR TITLE
app-worker Icinga check improvements

### PR DESCRIPTION
**Pull request by cbaines**
These changes should make the procfile worker checks clearer, and start monitoring the memory usage.

### CODE REVIEW

File: modules/govuk/manifests/apps/whitehall.pp [hunk 1]
```diff
@@ -318,9 +318,11 @@
...
 
     govuk::procfile::worker { 'whitehall-admin':
-      setenv_as      => $app_name,
-      enable_service => $enable_procfile_worker,
-      process_count  => $procfile_worker_process_count,
+      setenv_as                 => $app_name,
+      enable_service            => $enable_procfile_worker,
+      process_count             => $procfile_worker_process_count,
+      memory_warning_threshold  => 2000,
+      memory_critical_threshold => 14000,
```
Comment (barrucadu) [inline modules/govuk/manifests/apps/whitehall.pp]: That seems a pretty wide range, is this right?

Comment (cbaines) [reply modules/govuk/manifests/apps/whitehall.pp]: The memory usage varies similarly. The machines have enough for Sidekiq to use 14GB of memory without too much of an issue, which is why I set the critical threshold here, but generally, I was setting the critical threshold quite high.

Comment (bevanloon) [reply modules/govuk/manifests/apps/whitehall.pp]: @barrucadu - are you happy with this or do you still think we need more tweaking to the values?

Comment (barrucadu) [reply modules/govuk/manifests/apps/whitehall.pp]: Sorry for missing this; I'm happy with it.

**Pull request merged**